### PR TITLE
storage: _add_measures can handle multiple aggregations at the same time

### PR DESCRIPTION
**Pull request by jd**
This allows to batch storage and deletion of splits for a metric.

### CODE REVIEW

**Comment by jd** [comment]
Pull-request updated, HEAD is now 030970262ef57bbfa0fd9bac2185b6a81980db4c

**Comment by jd** [comment]
Pull-request updated, HEAD is now 6a199f9edfcdbaf3750f119f07a097352e36eded

**Comment by jd** [comment]
Fix comment

**Comment by jd** [comment]
Pull-request updated, HEAD is now aaf5375875a56547bc643050476f1609ef25dfed

File: gnocchi/storage/__init__.py [hunk 1]
```diff
@@ -329,118 +331,117 @@ def _store_timeserie_splits(self, metric, keys_and_splits,
         # stored in the case that we need to rewrite them fully.
         # First, fetch all those existing splits.
         existing_data = self._get_splits_and_unserialize(
-            metric, [(key, aggregation) for key in keys_to_rewrite])
+            metric, [(key, split.aggregation)
```
Comment (chungg) [inline gnocchi/storage/__init__.py]: if the split has the aggregation already, why we passing in the aggregation?

Comment (jd) [reply gnocchi/storage/__init__.py]: The answer is line 367 ;)

Comment (chungg) [reply gnocchi/storage/__init__.py]: oh i mean that aggregation is never used anywhere in this method. it only ever uses split.aggregation.

Comment (jd) [reply gnocchi/storage/__init__.py]: Again, aggregation is only passed because it's used as the key of the dict as explained in the comment L367.

Comment (chungg) [reply gnocchi/storage/__init__.py]: oh. it's from the other method.

**Pull request merged**